rclpy/src/rclpy/service.cpp

```diff
@@ -107,7 +107,16 @@ Service::service_send_response(py::object pyresponse, rmw_request_id_t * header)
 
   rcl_ret_t ret = rcl_send_response(rcl_service_.get(), header, raw_ros_response.get());
   if (RCL_RET_OK != ret) {
-    throw RCLError("failed to send response");
+    if (RCL_RET_TIMEOUT) {
```

Comment:
Shouldn't this be `if (RCL_RET_TIMEOUT == ret)`?

Reply:
Ah 😓 sorry my bad... thanks for pointing out! i will address it with other comments.

Reply:
addressed by https://github.com/ros2/rclpy/pull/1136/commits/da0947e957096c292e7810332955d5dae606a050